visualization/app/codeCharta/ui/mapTreeView/mapTreeViewLevelComponent.spec.ts

```diff
@@ -211,21 +213,25 @@ describe("MapTreeViewLevelController", () => {
 
         it("Subscribe Hover", () => {
 
+            instantiateModule("app.codeCharta.ui.mapTreeView"); 
```

Comment:
@alschmut @FernandoES  the problem was that in order to test things that rely on the event system of angular you need to use the "real" $rootScope or a mock that handles the events correctly. As long as $rootScope is just a "dumb" mock with no real event system implementation it cannot broadcast events. 
The solution here is to instatiate the module and get the real rootScope:)